pkg/ocm/managedservice.go

```diff
@@ -45,6 +46,13 @@ func (c *Client) CreateManagedService(args CreateManagedServiceArgs) (*msv1.Mana
 			msv1.NewServiceParameter().ID(id).Value(val))
 	}
 
+	var listeningMethod msv1.ListeningMethod
+	if args.Privatelink {
+		listeningMethod = msv1.ListeningMethodInternal
+	} else {
+		listeningMethod = msv1.ListeningMethodExternal
```

Comment:
do we want to handle this here? or should the cli not pass the `Listening` option at all to MS if it wasn't specified?

Reply:
Just to confirm, this will effectively set `Listening(ListeningMethodExternal)` for all clusters that don't specify the `PrivateLink` flag explicitly. Is this the correct default behavior? If so, I'm ok with merging this PR

Reply:
Marking this as resolved since we confirmed this in slack!